app/contributing/markdown-rules.md

```diff
@@ -96,11 +96,31 @@ For example, if the project path is `app/enterprise/2.1.x/overview`, the path in
 the nav file would be `/overview`, and you would add it to the file
 `app/_data/docs_nav_ee_2.1.x.yml`.
 
+### Add redirects
+
+If you're making an organization change like updating page nesting or renaming a top-level 
+menu item, you'll need to set up a redirect. Redirects prevent `404` pages, and 
+redirect users automatically to the new content location.
+
+1. Navigate to `app` then `_redirects`.
+2. Find the section of the documentation for which the redirect applies. For example, Dev Portal.
+3. Add a new line with the link you want to redirect **from**.
+4. On the same line, add the link you want to redirect **to**.
+
```

Comment:
I was also going to add a step 5 to "change all internal links to the new link" but wasn't sure it fit - the steps are for adding a redirect, not for changing a link. So, up to you if you think it makes sense to add a step here.

Reply:
Also chatting, we agreed that the important note is enough.